packages/api/.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@soumyadip007 this file could be removed

Reply:
Thanks, will add that in gitignore too